surveys/2022-annual-survey/questions.md

```diff
@@ -336,6 +336,36 @@ Type: free form (optional)
 > More detail on the type of situations where people have felt unwelcome can let us better 
 > address these issues in the future.
 
+## Rust in Education
+
+### Are you currently, or have you in the last year, taken a course or training which uses or teaches Rust?
+
+TODO should we combine these two questions?
```

Comment:
I think so - perhaps the following answers:

* Yes, through an educational institution
* Yes, through my employer
* No